docs/serving/autoscaling.md

```diff
@@ -179,11 +180,39 @@ template.
 spec:
   template:
     metadata:
-      autoscaling.knative.dev/metric: concurrency
+      autoscaling.knative.dev/metric: cpu
       autoscaling.knative.dev/class: hpa.autoscaling.knative.dev
 ```
 
-## Additional resources
+# Implementing your own Pod Autoscaler
+
+The Pod Autoscaler custom resource allows you to implement your own autoscaler without changing anything else about the Knative Serving system.
```

Comment:
"The Pod Autoscaler custom resource" sounds like its an preexisting object? 

Or is this task actually: "Create a custom resource to implement your own Pod Autoscaler"?

Reply:
Nope, this is not supposed to be "Create a custom resource". The resource is indeed pre-existing. We only create a different reconciler for it.